e2e/tests/modules/i18n/ui-language.e2e.js

```diff
@@ -0,0 +1,32 @@
+import { select, setupDriver, stopDriver } from '../../../driver/driver'
+import LandingPage from '../../../pages/routes/landing/LandingPage'
+
+describe('UI language', () => {
+  it('should match the system language de', async () => {
+    const driver = await setupDriver(select({
+      android: {language: 'de', locale: 'DE'},
+      ios: {language: 'de', locale: 'de_DE'}
+    }))
+
+    const landingPage = new LandingPage(driver)
+    await landingPage.ready()
+
+    expect(await landingPage.getSearchInput().text()).toBe('Suche nach deinem Ort')
```

Comment:
I don't really like comparing strings here. IMO the a test should not fail only because I changed something in the `locales.json`. Same below.

Reply:
I agree but there is not really a other way to do this. E2E tests can fail if you change the visuals of the app. I don't think we should make 100% sure they are as stable as unit tests. Also E2E tests should not depend on the source code imo. So there is not really a way to change that.